src/components/BottomNavBar.tsx

```diff
@@ -1,33 +1,76 @@
 import { NavLink } from 'react-router-dom';
-import discover from '../assets/nav-icon--discover.svg'
-import home from '../assets/nav-icon--home.svg'
-import chart from '../assets/nav-icon--chart.svg'
```

Comment:
why?

Reply:
I didn't know how to use filter method img elements so I've changed them into svg's and used fill property in css to change their color

Reply:
Would you mind on changing it back and using filter property? Converter link :
https://isotropic.co/tool/hex-color-to-css-filter/